README.md

```diff
@@ -1,6 +1,17 @@
 # Budget Rails app
 
 This project is for developing a budget app using rails
+
+- [Live Demo - Heroku](https://intense-harbor-49660.herokuapp.com/)
+- [Video Presentation](https://www.loom.com/share/6d01e431f7a0467dad0c6122f6cbd497)

```

Comment:
In your loom video, you didn't talk about the purpose of the project and about a piece of code you found interesting or proud of making, please redo another video and submit showing a clear and objective description about the project.